ingestion/src/metadata/ingestion/source/sql_source.py

```diff
@@ -329,17 +367,15 @@ def fetch_views(
                 except NotImplementedError:
                     view_definition = ""
 
-                description = _get_table_description(schema, view_name, inspector)
-                table_columns = self._get_columns(schema, view_name, inspector)
-                view_name = view_name.replace(".", "_DOT_")
-                fqn = f"{self.config.service_name}.{self.config.database}.{schema}.{view_name}"
                 table = Table(
                     id=uuid.uuid4(),
-                    name=view_name,
+                    name=view_name.replace(".", "_DOT_"),
                     tableType="View",
-                    description=description if description is not None else " ",
-                    fullyQualifiedName=fqn,
-                    columns=table_columns,
+                    description=_get_table_description(schema, view_name, inspector)
+                    or "",
+                    # This will be generated in the backend!! #1673
+                    fullyQualifiedName=view_name,
```

Comment:
Note https://github.com/open-metadata/OpenMetadata/issues/1673. We are instantiating Entities during the source (not CreateEntities). We had some non-needed logic for the FQDN which has been simplified, as this is going to happen at the server-side.